src/oic/oic/provider.py

```diff
@@ -2193,23 +2197,22 @@ def end_session_endpoint(
             try:
                 _ruri = self.cdb[client_id]["post_logout_redirect_uris"]
             except KeyError:
-                msg = "Missing post_logout_redirect_uri"
-                return error_response("invalid_request", msg)
-
-            if len(_ruri) == 1:
-                _base, _query = _ruri[0]
-                if _query:
-                    query_string = urlencode(
-                        [(key, v) for key in _query for v in _query[key]]
-                    )
-                    redirect_uri = "%s?%s" % (_base, query_string)
-                else:
-                    redirect_uri = _base
+                redirect_uri = self.post_logout_page
```

Comment:
This defaults to an empty string, what happens if it is not configured?

Reply:
That's a good question. That would probably lead to the user ending up with an error page.
What would you have happen ?

Reply:
Not sure either. Maybe change the default to None so it is easier to detect. Then test here if it is None, log a warning for the provider and display a simple page with an error?

Reply:
Could do that. At least it's something for the time being.

Reply:
Though that simple page has to reside somewhere. Not sure we can make any assumption on a reasonable URL.

Reply:
True... So maybe return an error response to the user?

Reply:
Yeah, let's go for that.